lib/ansible/modules/system/aix_chsec.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aix_chsec
+
+short_description: modify aix stanza files
+
+version_added: "0.1"
```

Comment:
```suggestion
version_added: '2.8'
```
This is a required fix as it states the first version this module ships with.